configure.ac

```diff
@@ -308,6 +310,22 @@ ACX_PTHREAD([
              CXX="$PTHREAD_CXX"],
             [AC_MSG_ERROR([Missing pthread, please install it])])
 
+if test -z "${USING_WIN32_TRUE}"; then
```

Comment:
It probably makes sense to extend https://github.com/OpenLightingProject/ola/blob/master/config/acx_pthread.m4